src/test/resources/feature/checkout2.feature

```diff
@@ -0,0 +1,69 @@
+Feature: complete order
+
+  Background: User is logged in
+
+  Background: User has items in the cart
+
+  Scenario: Verify that user can complete order with default address
+    Given User is on shopping page
+    When checkout button is clicked
+    Then Summary page is displayed
+    And Products are displayed
+    When Proceed to Checkout button is clicked
+    Then Addresses page is displayed
+    And Default Address is displayed
+    When Proceed to Checkout button is clicked
+    Then Shipping page is displayed
+    When Terms of Service checkbox is clicked
+    And Proceed to Checkout button is clicked
+    Then Payment page is displayed
+    When Payment method Pay by wire is clicked
+    Then Order Summary page is displayed
+    When I Confirm My Order button is clicked
+    Then Order Confirmation page is displayed
+    And Amount is displayed
+    When Back To Orders button is clicked
+    Then Order History page is displayed
+    When Order reference is clicked
+    Then FOLLOW YOUR ORDER section is displayed
+    And Delivery Address is default address
+
+
+  Scenario: Verify that user can complete order with new address
+    Given User is on shopping page
+    When checkout button is clicked
+    Then Summary page is displayed
+    And Products are displayed
+    When Proceed to Checkout button is clicked
+    Then Addresses page is displayed
+    And Default Address is displayed
+    When Add New Address button is clicked
+    Then Your Address page is displayed
+    When <First name> is entered in First name text box
+    And <Last name> is completed Last name text box
+    And <Address> is completed Address text box
+    And <City> is completed City text box
+    And <State> is selected State dropdown
+    And <Phone> is completed Phone text box
+    And <Address title> is completed Address title text box
+    And Proceed to Checkout button is clicked
+    Then Shipping page is displayed
+    When Terms of Service checkbox is clicked
+    And Proceed to Checkout button is clicked
+    Then Payment page is displayed
+    When Payment method Pay by wire is clicked
+    Then Order Summary page is displayed
+    When I Confirm My Order button is clicked
+    Then Order Confirmation page is displayed
+    And Amount is displayed
+    When Back To Orders button is clicked
+    Then Order History page is displayed
+    When Order reference is clicked
+    Then FOLLOW YOUR ORDER section is displayed
+    And Delivery Address is <new Address>
+
+  Scenario Outline:
```

Comment:
Could you please remove Scenario Outline from here and change "**Scenario**: Verify that user can complete order with new address: to **Scenario Outline**.

Here is an example of Scenario Outline:

```
Scenario Outline: eating
  Given there are <start> cucumbers
  When I eat <eat> cucumbers
  Then I should have <left> cucumbers

  Examples:
    | start | eat | left |
    |    12 |   5  |    7  |
    |    20 |   5  |   15 |
```

Reply:
Done